src/pineko/evolve.py

```diff
@@ -113,11 +116,21 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     xif = 1.0 if sv_method is not None else tcard["XIF"]
     operators_card["configs"]["scvar_method"] = sv_method
     q2_grid = (xif * xif * muf2_grid).tolist()
-    operators_card["_mugrid"] = np.sqrt(q2_grid).tolist()
+    atlas = ThresholdsAtlas(
+        masses=np.array([tcard["mc"], tcard["mb"], tcard["mt"]]) ** 2,
+        q2_ref=tcard["Q0"],
```

Comment:
```suggestion
        q2_ref=tcard["Q0"] ** 2,
```
@andreab1997 I'm not sure if this means I screwed some ekos...

Reply:
Yes, but I believe this affects everything I fear

Reply:
Ah no, I am sorry, you are right. It should only affect the points between `1.65` and `1.65**2` which we do not fit.